web/src/people.ts

```diff
@@ -41,7 +41,11 @@ export type User = {
     profile_data: Record<number, ProfileData>;
     is_missing_server_data?: boolean; // used for fake user objects.
     is_inaccessible_user?: boolean; // used for inaccessible user objects.
+    // Used in `typeahead_helper`
+    special_item_text?: string;
 
+    // Used to differentiate from PseudoMentionUser in `typeahead_helper`
+    is_broadcast: false;
```

Comment:
These fields also feel like they belong to some type generated from the `User` type, not the `User` type itself.

Reply:
Yup that's the one remaining task I have left for this PR, to fix that up